docs/reference/changelog.md

```diff
@@ -15,7 +15,7 @@
 * [#422](https://github.com/open-contracting/infrastructure/pull/422), [#425](https://github.com/open-contracting/infrastructure/pull/425) - add CoST IDS sustainability modules to mapping documentation.
 
 ### Schema
-
+https://github.com/open-contracting/infrastructure/pull/445/commits/59f5b6b7c85d88e58bddb063fa8138e347817aac
```

Comment:
Bad merge? (Restore newline)

Reply:
An accidental paste, by the look of it. Fixed in https://github.com/open-contracting/infrastructure/pull/445/commits/d226f235aec4d1a226b2567885391d9553e35d3c. Thanks for flagging